src/components/DayPicker.jsx

```diff
@@ -825,6 +825,9 @@ export default class DayPicker extends React.Component {
       <div
         className={dayPickerClassNames}
         style={dayPickerStyle}
+        role="application"
```

Comment:
Added a role of application here as it seemed to match up with the WAI-ARIA example shown here: [For application landmarks with static prose](https://www.w3.org/TR/2009/WD-wai-aria-practices-20090224/#kbd_layout_static).